packages/e2e-test-utils/src/text-content-areas-have-focus.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { textContentAreas } from './text-content-areas';
+
+/**
+ * Tabs through the text content areas of a block and asserts the expected values
+ *
+ * @param {string} content The expected value of the block's contenteditable elements
+ * @return {Promise} A promise that's resolved when the browser has finished tabbing throught the contenteditable areas of a block, and asserting they have keyboard focus and the expected content.
+ */
+
+export async function textContentAreasHaveFocus( content ) {
+	const blocks = await textContentAreas( { empty: false } );
```

Comment:
How about
```suggestion
	const blockContentAreas = await textContentAreas( { empty: false } );
```
here as well?